content/doc/administration/requirements/linux.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: documentation
+title:  Linux Support Policy
+---
+
+This page documents the Linux support policy for the Jenkins controller and agents.
+
+== Scope
+
+Jenkins plugins may set additional requirements to Linux versions on controllers and/or agents.
+This page does not document such requirements.
+Please refer to plugin documentation.
+
+== Why?
+
+Theoretically, Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Linux API and subsystems,
+and hence they rely on specific Linux versions.
+Jenkins platform specific installation packages rely on specific Linux versions.
+
+== Support levels
+
+We define multiple support levels for Linux platforms.
+
+[width="100%",cols="20%,35%,45%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to fix the reported issues timely.
+a|
+  * 64-bit (amd64) Linux versions supported by Debian, Red Hat, SUSE, and Ubuntu
+  * Linux versions used in the official Docker images
+
+| **Level 2** - Supported
+| We do not actively test these platforms, but we intend to keep compatibility.
+  We will consider patches if they do not put Level 1 support at risk and do not create maintenance overhead.
+a|
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the Debian packaging format
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the rpm packaging format
```

Comment:
Packaging format has nothing to do with the supported platforms. What is the intent of excluding other platforms?

Reply:
I hadn't considered a user that chooses to use the war file rather than using a package manager.  I'm open to suggestions of alternate phrasing that would also cover users running with a war file.

Reply:
I've included phrasing for war file users and for container users.